lib/gollum/public/gollum/stylesheets/template.scss

```diff
@@ -727,3 +727,14 @@ a {
     }
   } 
 }
+
```

Comment:
Instead of adding CSS to `template.scss`, it would be preferable I think to use Primer.css classes to the pagination mustache template.

Reply:
(For example, using the class `float-right` on the span "next".)

Reply:
Done! Thanks for the tip, much cleaner like this.